app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -179,6 +179,19 @@ function FileInput(props, ref) {
     }
   }
 
+  /**
+   * @param {Blob|string|null|undefined} fileValue Optional array of file input accept patterns.
+   */
+  function getLabelFromValue(fileValue) {
+    if (fileValue instanceof String) {
```

Comment:
`fileValue` as a string is a [data URL](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) of the image contents, which we won't want to set as the `aria-label`. Thinking maybe we can only get the `name` from a File, then anything else truthy we use a fallback.

```js
if (fileValue instanceof window.File) {
  return fileValue.name;
} else if (fileValue) {
  return 'Captured image'; // TODO: Translate
}
```

Since the input will announce "no file selected" without a label, maybe that's what we'd want if `fileValue` was empty? i.e. not return anything here.